funnel/views/project.py

```diff
@@ -485,6 +496,29 @@ def settings(self):
             'project_save_form': project_save_form,
         }
 
+    @route('comments', methods=['GET', 'POST'])
+    @render_with('project_comments.html.jinja2')
+    @requires_login
+    @requires_roles({'reader'})
+    def project_comments(self):
+        project_save_form = SavedProjectForm()
+        comments = sorted(
+            Comment.query.filter_by(commentset=self.obj.commentset, parent=None)
+            .order_by('created_at')
+            .all(),
+            key=lambda c: c.voteset.count,
```

Comment:
Comment votes are currently hidden in the UI. Has this changed? In any case, sorting can be pushed to the db by using a join on `Voteset` and `order_by(Voteset.score, Comment.created_at.desc())` (not `Vote.count`).

Reply:
`Voteset` doesn't have any column or property called `score`.